pom.xml

```diff
@@ -89,6 +89,12 @@
       <version>${mockito.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
It looks like `guava` is still here. I am happy to remove this dependency and move this PR through... unless you would prefer to add a commit.